main.qml

```diff
@@ -23,8 +28,41 @@ Window {
     {
         id: _controller
         Component.onCompleted: {
-            _board.move.connect(_controller.makeMove);
+            _board.move.connect(root.move);
+            _statusBar.reset()
+            _controller.winner.connect(root.showWinScreen)
+            _victory.newGame.connect(root.newGame)
         }
+
+
+    }
+
+    function showWinScreen() {
+        _victory.open()
+    }
+
+    function move(index) {
+        if (_controller.makeMove(index))
```

Comment:
What about the connection between these things? For example, if move succeeds, this function will be called automatically